File: 6 kyu/Counting Duplicates/solution.py
def duplicate_count(text: str) -> int:
    text: str = text.lower()
    return sum(True for x in set(text) if text.count(x) > 1)
  
_______________
def duplicate_count(s):
  return len([c for c in set(s.lower()) if s.lower().count(c)>1])

_______________
def duplicate_count(text):
    seen = set()
    dupes = set()
    for char in text:
        char = char.lower()
        if char in seen:
            dupes.add(char)
        seen.add(char)
    return len(dupes)
  
_________________
from collections import Counter

def duplicate_count(text):
    return sum(1 for c, n in Counter(text.lower()).iteritems() if n > 1)
  
_________________
def duplicate_count(text):
    count = 0
    for c in set(text.lower()):
        if text.lower().count(c) > 1:
            count += 1
    return count
     
__________________
from collections import Counter

def duplicate_count(text):
    counter = Counter(text.lower())
    return len([counter.keys() for i in counter.values() if i>1])
  
_______________
def duplicate_count(text):
    text=text.lower()
    x=[ text.count(x) for x in set(text) ]
    y=x
    x=[ i  for i in x if i !=1  ]
    return len(x)
  
______________
def duplicate_count(text):
    char_count_dict = {}
    duplicated_chars = []
    for char in text.lower():
        char_count = char_count_dict.get(char, 0)
        char_count += 1
        char_count_dict[char] = char_count
        
        if char_count > 1:
            duplicated_chars.append(char)
             
    return len(set(duplicated_chars))

File: 6 kyu/Counting Duplicates/solution.go
package kata
import "strings"
func duplicate_count(s string) (c int) {
    h := map[rune]int{}
    for _, r := range strings.ToLower(s) {
      if h[r]++; h[r] == 2 { c++ }
    }
    return
}
________________________
package kata

import "strings"

func duplicate_count(s1 string) (c int) {
    m := make(map[rune]int)
    for _,v := range strings.ToLower(s1) {
        m[v]++
        if m[v] == 2 { c++ }
    }      
    return
}
________________________
package kata

import "strings"

func duplicate_count(s1 string) int {
    counter := make(map[string]int)

    for _, s := range s1 {
      sl := strings.ToLower(string(s))
      counter[sl] += 1
    }

    sum := 0
    for _, v := range counter {
      if v > 1 {
        sum += 1
      }
    }

    return sum
}
_____________________
package kata

import "strings"

func duplicate_count(s1 string) (repeatCount int) {
  counters := make(map[rune]int)
  
  for _, letter := range strings.ToLower(s1) {
    if counters[letter] == 1 {
      repeatCount++
    }
    counters[letter]++
  }
  
  return
}
______________________
package kata

import "strings"

func duplicate_count(s1 string) int {
    var m map[byte] int
    var doubles int = 0
    
    m = make(map[byte]int)
    
    var lower string = strings.ToLower(s1)
    
    for i := 0; i < len(lower); i++ {
      m[lower[i]] += 1
      
      if m[lower[i]] == 2 {
        doubles += 1
      } 
    }
    
    return doubles
    
}
